spec/yearmonth.html

```diff
@@ -385,6 +385,19 @@ <h1>BalanceYearMonth ( _year_, _month_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-constraindate" aoid="ConstrainDate">
+      <h1>ConstrainYearMonth ( _year_, _month_ )</h1>
+      <emu-alg>
+        1. Assert: _year_ and _month_ are integer Number values.
+        1. Set _year_ to ConstrainToRange(_year_, -999999, 999999).
```

Comment:
I think these need a `!` for "assert no abrupt completion"? Or is that optional when it's just a mathematical function?

Reply:
At the moment, we prefer the prefix on every abstract operation call.